frontend/packages/console-app/src/plugin.tsx

```diff
@@ -11,8 +11,8 @@ const plugin: Plugin<ConsumedExtensions> = [
       id: 'admin',
       name: 'Administrator',
       icon: <CogsIcon />,
-      landingPageURL: '/k8s/cluster/projects',
-      k8sLandingPageURL: '/cluster-status',
+      landingPageURL: '/dashboards',
```

Comment:
We can't change this for all users. We need to make RBAC checks. Most users can't view this page.